manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -106,6 +106,32 @@ metadata:
     exclude.release.openshift.io/internal-openshift-hosted: "true"
 spec:
   groups:
+  - name: aggregated-apis
+    rules:
+    - alert: AggregatedAPIErrors
+      annotations:
+        message: "An aggregated API {{ $labels.name }}/{{ $labels.namespace }} have reported errors."
+        description: "The number of error ({{ $labels.reason }}) has increased for an aggregated API {{ $labels.name }}/{{ $labels.namespace }} in the last 2 minutes."
+      expr: |
+        sum by(name, namespace, reason)(increase(aggregator_unavailable_apiservice_count[2m])) > 1
+      labels:
+        severity: warning
+    - alert: AggregatedAPIDown
+      annotations:
+        message: "An aggregated API {{ $labels.name }}/{{ $labels.namespace }} down."
+        description: "An aggregated API {{ $labels.name }}/{{ $labels.namespace }} has not been available for 1 min."
+      expr: |
+        sum by(name, namespace)(sum_over_time(aggregator_unavailable_apiservice[1m])) > 0
+      labels:
+        severity: warning
+    - alert: AggregatedAPIFlapping
+      annotations:
+        message: "An aggregated API {{ $labels.name }}/{{ $labels.namespace }} flapping."
+        description: "An aggregated API {{ $labels.name }}/{{ $labels.namespace }} availability changes too often. Seen at least 2 changes in the last 5 minutes."
+      expr: |
+        sum by(name, namespace)(changes(aggregator_unavailable_apiservice[5m])) > 2
```

Comment:
it deserves an e2e test that would help find good values.

Reply:
also is not perfect, frequent changes can be unnoticeable  - value hasn't changed between the scrapes.

Reply:
longer duration might give a better perspective.

Reply:
alternatively, we could use `sum by(name, namespace, reason)(changes(aggregator_unavailable_apiservice_count[5m])) > 1`